.github/workflows/_windows_ut.yml

```diff
@@ -114,7 +114,7 @@ jobs:
           pip install cmake setuptools==72.1.0 clang-format
           pip install mkl-static==2025.0.1 mkl-include==2025.0.1
           set USE_KINETO=0
-          set USE_ONEMKL=1
+          set USE_ONEMKL_XPU=1
```

Comment:
Please remove it if it is default on by design.

Reply:
Done.